content/data-reporting/changelog/index.mdx

```diff
@@ -0,0 +1,18 @@
+---
+weight: 30
+category: "data-reporting"
+subcategory: "changelog"
+title: "Changelog"
+tagline: "Changelog of the Reports API"
+desc: "Changelog of the Reports API"
+---
+
+This document is the record of all the changes in [**the Reports API**](/data-reporting/reports-api/) starting from **version 3.2**.
+
+## [v3.3] - Developer preview
```

Comment:
We'll probably remove this line if there are no changes by the time we publish the docs.